src/utils/translationUtils.js

```diff
@@ -291,22 +291,15 @@ export function translateSchema(schemaJson, schemaType) {
   return schema
 }
 
-export function translateListingCategory(listingObj) {
-  // Copy the listing so we don't modify the original
-  const listing = JSON.parse(JSON.stringify(listingObj))
-  const category = listing.category
-
-  // Check to see if category is a react-intl ID
-  if (/schema\./.test(category)) {
-    // loop over all schemaMessages to find the correct ID
-    for (const schemaType in schemaMessages) {
-      if (schemaMessages[schemaType][category]) {
-        listing.category = globalIntlProvider.formatMessage(
-          schemaMessages[schemaType][category]
-        )
-      }
-    }
+export function translateListingCategory(rawCategory) {
+  const match = rawCategory.match(/^schema\.([^.]+)\.([^.]+)$/)
+  if (match === null) {
+    return rawCategory
   }
-
-  return listing
+  const [_, schemaType, category] = match
```

Comment:
I'm with you, it is so irritating we can't use _ for unused variables.
But the build is failing due to eslint complaining:
  299:10  error  '_' is assigned a value but never used         no-unused-vars
  299:25  error  'category' is assigned a value but never used  no-unused-vars

Reply:
I didn't realize at first that the dapp had the formatting/lint check under npm test. I'll run that in the future.